### PR TITLE
R9M Lite Pro DAC Outputs

### PR DESCRIPTION
**Pull request by Jahnkeanater**
This "should" be enough to get the Lite Pro working but I'm sure there is a better way to do it.

I'm not sure what the function Radio.SetOutputPower is actually doing. I think its the actual power output of the SX1276?

I couldn't find where GPIO_PIN_RFamp_APC1 or GPIO_PIN_RFamp_APC2 were actually used other than the R9M's I2C DAC lib.

#326

### CODE REVIEW

File: src/src/tx_main.cpp [hunk 1]
```diff
@@ -541,6 +541,13 @@ void setup()
     pinMode(GPIO_PIN_LED_GREEN, OUTPUT);
     pinMode(GPIO_PIN_LED_RED, OUTPUT);
     digitalWrite(GPIO_PIN_LED_GREEN, HIGH);
+  
+#if defined(TARGET_R9M_LITE_PRO_TX)
+//#define HAL_DAC_MODULE_ENABLED //this may need to be enabled in the Arduino stm32h7xx_hal_conf.h file
+pinMode(GPIO_PIN_RFamp_APC1, OUTPUT); //I'm sure this does not belong here
```
Comment (cruwaller) [inline src/src/tx_main.cpp:547]: I think better place for these is in `void POWERMGNT::init()`

File: src/lib/POWERMGNT/POWERMGNT.h [hunk 2]
```diff
@@ -20,8 +20,8 @@
 #endif
 
 #ifdef TARGET_R9M_LITE_PRO_TX
-#define MaxPower 0
-#define DefaultPowerEnum 0
+#define MaxPower 6
```
Comment (cruwaller) [inline src/lib/POWERMGNT/POWERMGNT.h:23]: Use those enumerations here. Other targets need to be aligned as well :) 
`#define MaxPower PWR_1000mW` and `#define DefaultPowerEnum PWR_100mW` .

Comment (Jahnkeanater) [reply src/lib/POWERMGNT/POWERMGNT.h:23]: It doesn't look like the values of PowerLevels_e are consistent for different targets.

**Review by cruwaller — APPROVED** [review]
Thanks!

File: src/lib/POWERMGNT/POWERMGNT.h (hunk 2 above)
Comment (cruwaller) [reply src/lib/POWERMGNT/POWERMGNT.h]: I know, you changed all... Thanks a lot!

**Comment by JyeSmith** [comment]
Is this PR still needed after merging 366?

https://github.com/ExpressLRS/ExpressLRS/pull/366

**Comment by Jahnkeanater** [comment]
> 
> 
> Is this PR still needed after merging 366?
> 
> #366

I don't think so. I'm pretty sure VladimirP1 included this in his pull request.

**Pull request closed without merging**